CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(OpenCLHeaders
+  VERSION 2.2
```

Comment:
Should this be 3.0? We're using a date scheme for tags. What should we use here?

Reply:
Because we got shapeshifting headers based on defines, I see two meaningful choices:
- Maximum supported version
- Date tag

The question to ask here: what would users want to test for? The availability of an API version in the headers, or a fix to some bug? Both checks are likely made in build systems. IMHO we should offload this question to a separate issue, as we've already have #94 as an anchor for this. I vote for a separate PR, a oneliner even.

Reply:
Yeah, agree this needs a bit of discussion. I think using the maximum supported version is the most likely outcome. We could bump this to 3.0 and defer the full discussion to a separate issue.